docs/src/main/sphinx/admin/properties-node-scheduler.rst

```diff
@@ -30,21 +30,38 @@ due to splits not being balanced across workers. Ideally, it should be set
 such that there is always at least one split waiting to be processed, but
 not higher.
 
-``node-scheduler.max-pending-splits-per-task``
+``node-scheduler.min-pending-splits-per-task``
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
 * **Type:** :ref:`prop-type-integer`
 * **Default value:** ``10``
 
-The number of outstanding splits with the standard split weight that can be
+The maximum number of outstanding splits with the standard split weight that can be
```

Comment:
`The minimum number of outstanding splits with the standard split weight guaranteed to be scheduled on a node (even when the node is already at the limit for total number of splits) for a single task given the task has remaining splits to process. Allowing a minimum number of splits per stage is required to prevent starvation and deadlocks.`
EOL